modulefiles/compiler/compilerName/compilerVersion/metplus/metplus.lua

```diff
@@ -0,0 +1,39 @@
+help([[
+]])
+
+local pkgName    = myModuleName()
+local pkgVersion = myModuleVersion()
+local pkgNameVer = myModuleFullName()
+
+local hierA        = hierarchyA(pkgNameVer,1)
+local compNameVer  = hierA[1]
+local compNameVerD = compNameVer:gsub("/","-")
+
+conflict(pkgName)
+
+always_load("ips")
+always_load("python/3.6.3")
+always_load("nco")
+always_load("grib_util")
+always_load("met")
+prereq("ips")
+prereq("python/3.6.3")
```

Comment:
These are specific to WCOSS_D.  Please remove.
The modulefile is already in the hierarchy of a compiler.  It is not available until a compiler module is loaded.

Reply:
Does MetPlus depend on these?
I did not see module load's in the metplus build script.